pkg/virt-controller/services/template_test.go

```diff
@@ -53,7 +53,11 @@ var _ = Describe("Template", func() {
 				}))
 				Expect(pod.ObjectMeta.GenerateName).To(Equal("virt-launcher-testvm-----"))
 				Expect(pod.Spec.NodeSelector).To(BeEmpty())
-				Expect(pod.Spec.Containers[0].Command).To(Equal([]string{"/virt-launcher", "-qemu-timeout", "60s"}))
+				Expect(pod.Spec.Containers[0].Command).To(Equal([]string{"/virt-launcher",
+					"-qemu-timeout", "60s",
+					"-name", "testvm",
+					"-namespace", "testns",
+					"-socket-dir", "/var/run/libvirt"}))
```

Comment:
In the normal runtime, socket-dir is always set to /var/run/kubevirt, but for some reason all the tests set it to /var/run/libvirt.  Is this intentional?